packages/google_maps_flutter/google_maps_flutter/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.0.4
+
+* Minor cleanup of Android code.
```

Comment:
nit: call out the specific things that were done (especially the leak fix).

Reply:
Done